homeassistant/components/person/__init__.py

```diff
@@ -404,12 +404,17 @@ async def async_reload_yaml(call: ServiceCall) -> None:
     return True
 
 
-class Person(collection.CollectionEntity, RestoreEntity):
+class Person(
+    collection.CollectionEntity,
+    RestoreEntity,
+):
     """Represent a tracked person."""
 
     _entity_component_unrecorded_attributes = frozenset({ATTR_DEVICE_TRACKERS})
 
     _attr_should_poll = False
+    _attr_state: str | None = None
+    device_trackers: list[str] = []
```

Comment:
We shouldn't initialize a class attribute to a mutable object. That's error prone. Please move it to the init method.